### PR TITLE
Attempt to add ReactJSON view to rendered templates

### PR DESCRIPTION
**Pull request by jscheffl**
This PR makes a nicer rendering of rendered template values - if they are kind of JSON.

Details view would be like:
image

### CODE REVIEW

File: airflow/www/static/js/dag/details/taskInstance/Details.tsx [hunk 1]
```diff
@@ -335,11 +351,49 @@ const Details = ({ gridInstance, taskInstance, group }: Props) => {
                       // skip
                     }
                   }
+                  const fieldStr = field as string;
+                  try {
+                    const fieldJson = JSON.parse(fieldStr);
+                    return (
```
Comment (bbovenzi) [inline airflow/www/static/js/dag/details/taskInstance/Details.tsx]: Oh let's not return inside of the `try/catch` block. Instead just assign the fieldJson variable inside try/catch and then add a check if its not null inside the jsx return

Comment (bbovenzi) [reply airflow/www/static/js/dag/details/taskInstance/Details.tsx]: This is a lot of logic now. How about we make a separate `RenderedFields` component and handle all of this logic there?

Comment (jscheffl) [reply airflow/www/static/js/dag/details/taskInstance/Details.tsx]: Thanks for the hints, needed ~10 iterations but now I assume I got it. Consolidated and in total only 19 lines added in code-base for a common component - XCom following :-D

**Review by bbovenzi — APPROVED** [review]
nice work!

**Pull request merged**